### DISCUSSION TITLE
为各位绳匠提供入门款头像，欢迎使用，一共八张

### DISCUSSION THREAD

**Discussion by clcl-maker** [share121/inter-knot#3341] [其他]
头像来源于网络，大家自行使用。请共同维护社区环境！！谢谢！！
![6ecba311563ed476931edb5b5bb5f394](https://github.com/user-attachments/assets/30693251-c2c3-48d9-ab68-b42e51c57bf0)
![9cf78e666f1cda3bc150d4278fd8a4ad](https://github.com/user-attachments/assets/cf3140a7-e7f0-46b0-9bf7-38cc5612c8b3)
![21879c8d2019c49d2aad74ddf1d4e8d1](https://github.com/user-attachments/assets/d62891ea-60b5-43b1-8726-c673d1612047)
![825048b3fd8cdfb1ba1c2bb14e62bf1e](https://github.com/user-attachments/assets/c0617dbd-3988-407b-9f39-0b9153fd7bd4)
![f9e4ddf9451c499a6c36bbb5f95bedf6](https://github.com/user-attachments/assets/7a0273a5-03d8-4a02-9daa-abba9cdcda14)
![f344c922a087c633eb026f05041101be](https://github.com/user-attachments/assets/86f17a9a-1da5-4424-ada7-4286b248bb82)
![fc036cc78a6ca728ca62634de4e1523e](https://github.com/user-attachments/assets/ad46cdda-eb59-432d-834f-97c86633c65c)
![QQ图片20240820220351](https://github.com/user-attachments/assets/bdf44652-55e8-403c-960f-ed2480fe7a9b)
![QQ图片20240820221206](https://github.com/user-attachments/assets/6a12a29c-32fc-4695-b9e9-9803805fe00c)

**Comment by clcl-maker** [comment]
![6ecba311563ed476931edb5b5bb5f394](https://github.com/user-attachments/assets/523d4aa4-e90c-44ac-8b0a-77895ecc038f)
补充一张

**Comment by stormcat-efforia** [comment]
谢谢大佬，好人一生平安

**Comment by SKWTEA** [comment]
楼主大好人！

**Comment by wyqsagiri** [comment]
感谢楼主大大！好人一生平安！
